oaknational/Oak-Web-Application
# [CUR-453] Added fieldset/legend to group thread/year radio buttons in curriculum unit sequence page

## Description
Added `<fieldset/>`/`<legend/>` to group thread/year radio buttons in curriculum unit sequence page improving the accessibility.

We've also added a new `<SimpleRadio/>` as the existing radio input components don't play well with a `<fieldset/>`

## Issue(s)

`CUR-453`

## How to test

1. Go to https://deploy-preview-2504--oak-web-application.netlify.thenational.academy/teachers/curriculum/science-secondary-edexcel/units (any subject applicable)
2. Use voice over to check accessibility of the page
